test/markdown_test.dart

```diff
@@ -18,30 +18,46 @@ void main() {
 
   group('Resolver', () {
     Node nyanResolver(String text) => new Text('~=[,,_${text}_,,]:3');
-    validateCore('simple link resolver', '''
+    validateCore(
```

Comment:
Oof, yeah, it's ugly either way. Some of these should probably be moved to test data files like the other tests. Maybe all of them?